### DISCUSSION TITLE
ValueError('Namespace %s not available' % namespace) on Raspberry Pi

### DISCUSSION THREAD

**Discussion by akif1994** [aviolaris/fritzbox-tray#1] [Q&A] [answered]
Hello,

I've been using your app on a Windows PC for a few days now and I decided to give it a shot on my Raspberry Pi today but I got an error that said:

```
File "/usr/lib/python3/dist-packages/gi/init.py", line 126, in require_version
raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AyatanaAppIndicator3 not available
```

So I'm wondering if the app is compatible with ARM7 as I am unsure if this error is related to the hardware or something else.

Thank you for any answer (and for the app!).

**Comment by aviolaris** [comment, marked as answer]
Thank you for your feedback. This application is cross-platform, making it compatible with the ARM7 architecture. The aforementioned error message is a [common issue](https://github.com/aviolaris/fritzbox-tray#common-issues) that indicates that the `AppIndicator3` module, used by the application to display tray icons on Linux-based distributions, is not available on your system.

To resolve this issue, you will need to install the `libayatana-appindicator3-dev` package, which provides the necessary files and libraries for the `AppIndicator3` module. To install the package, you can use the following command:

    sudo apt-get install libayatana-appindicator3-dev

**Reply by akif1994** [reply]
Oh, my bad... I totally missed the common issues section. It worked after the installation. Thanks a bunch!
